tests/lsdb/core/test_kdtree_validation.py

```diff
@@ -31,7 +31,7 @@ def test_bounded_kdtree_radius_invalid(small_sky_catalog, small_sky_order1_sourc
 def test_kdtree_left_columns(small_sky_catalog, small_sky_order1_source_with_margin):
     original_df = small_sky_catalog._ddf
     small_sky_catalog._ddf = original_df.reset_index()
-    with pytest.raises(ValueError, match="index of left table must be _hipscat_index"):
+    with pytest.raises(ValueError, match="index of left table must be _healpix_29"):
```

Comment:
I've seen this in a few other places.  Out of curiosity, why not `_hats_index`?

Reply:
We want to be clearer that this is not a unique index to be used for primary keys.